manifest.php

```diff
@@ -28,7 +28,7 @@
     'name'        => 'taoQtiItem',
     'label'       => 'QTI item model',
     'license'     => 'GPL-2.0',
-    'version'     => '11.1.0',
+    'version'     => '11.1.1',
```

Comment:
it is more a feature addition, no ? so follow the semantic versionning, should it rather not be 11.2.0 instead ?